data/John Verhoefen (a commercial agent).txt

```diff
@@ -196,24 +197,22 @@ mission "John Verhoefen: Bankrupt [stage 1]"
 		"John Verhoefen: stage" = -1
 		"reputation: Merchant" -= 2
 	on complete
-		payment 100
 		# Merchants reputation increases and that depends on the number of given free rides (i.e. +1 for the first free ride, +2 for the 2nd free ride, ...)
 		"reputation: Merchant" += "John Verhoefen: free transfers"
-		"John Verhoefen: credits" -= <paymentNum>
```

Comment:
I had this fixed payment of 100 by purpose. Though he asked for a free ride and the player agreed, in the end (after/when arriving) he feels obliged to pay at least something.